3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/miscEnrichmentSpecs.scala

```diff
@@ -37,7 +37,7 @@ class EtlVersionSpec extends MutSpecification {
 
   "The ETL version" should {
     "be successfully returned" in {
-      MiscEnrichments.etlVersion("hadoop-0.14.0") must_== "hadoop-0.14.0-common-0.18.0"
+      MiscEnrichments.etlVersion("hadoop-1.4.0") must_== "hadoop-1.4.0-common-0.19.0"
```

Comment:
hey @alexanderdean, this change:

`hadoop-0.14.0` -> `hadoop-1.4.0`

I'm not sure what it means? I'm just fixing a conflict for R75 and want to make sure changes like this get percolated through properly